torch/_dynamo/variables/torch.py

```diff
@@ -750,6 +750,16 @@ def check_any_unspec(x):
                 else:
                     unimplemented(f"out variant of {type(kwargs['out'])}")
 
+            name = fn_.__name__
+            if (
+                not name.startswith("_")
+                and name.endswith("_")
```

Comment:
This is not a good way to test this. I think we should infer this by testing the fake tensor output when we fake prop to see if tensor identity was preserved.